zeebe/exporters/rdbms-exporter/src/main/java/io/camunda/exporter/rdbms/RdbmsExporterWrapper.java

```diff
@@ -158,13 +171,28 @@ private int readInt(final Context context, final String property, final int defa
     }
   }
 
-  private static void createHandlers(
+  private long readLong(final Context context, final String property, final long defaultValue) {
+    final var arguments = context.getConfiguration().getArguments();
+    if (arguments != null && arguments.containsKey(property)) {
+      return (Long) arguments.get(property);
+    } else {
+      return defaultValue;
+    }
+  }
+
+  private long readMaxCacheSize(final Context context) {
+    return readLong(context, "maxCacheSize", DEFAULT_MAX_CACHE_SIZE);
```

Comment:
🔧 Maybe we should specify which cache? I could image in the future we might have multiple caches.

`processCache.maxCacheSize`

💭 Off topic:
It would be nice to use an actual class for the config like in `CamundaExporter`: 
`configuration = context.getConfiguration().instantiate(ExporterConfiguration.class);` instead of parsing every property one by one. But this refactoring is definitely not in the scope of this ticket.
I'm not even sure which project should this belong to as a follow-up 🤔